addons/mail/static/src/components/chat_window/chat_window.scss

```diff
@@ -83,13 +54,8 @@
     }
 }
 
-.o_ChatWindow_header {
-    border-radius: 3px 3px 0 0;
-}
-
 .o_ChatWindow_newMessageFormInput {
     outline: none;
-    border: $border-width solid gray('300'); // cancel firefox border on input focus
 }
 
 .o_ChatWindow_thread .o_ThreadView_messageList {
```

Comment:
It sounds a bit off to have font-size for `MessageList` defined here :/
If it's a contextual value it should be probably be set using CSS variables instead.
@seb-odoo , any feedback on this?

Reply:
Indeed it's probably there for historical reasons. We should remove the specificity and add a conditional class in message list itself.

Reply:
Note: no need to do all changes here. This should be done in either a preparation PR, or follow up PR, depending on which is easier.